iac-extensions/kubernetes/src/test/resources/checks/CapabilitiesCheck/CapabilitiesChart/templates/capabilities-pod.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-caps1
+spec:
+  containers:
+    - image: ubuntu
+      command: ["sleep", "6000"]
+      name: test-container
+      securityContext:
+        capabilities:
+# Noncompliant@+1
+          add: {{ .Values.capabilities | toYaml | nindent 10 }}
+#                        ^^^^^^^^^^^^^^^^^^^^
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-caps1
```

Comment:
Is this test duplicating the first one? Should it be `capabilities2`?

Reply:
ok fixed names.